src/utils/check-pr/check-pr.py

```diff
@@ -188,9 +190,9 @@ def merge(self) -> None:
     def deploy(self, filename: str) -> None:
         print(f"deploying {filename} to {self.instance}")
         venv = "deploy-venv"
-        subprocess.check_call(f"python -mvenv {venv}", shell=True)
+        subprocess.check_call(f"python3 -mvenv {venv}", shell=True)
```

Comment:
this should stay as "python"

Reply:
Thanks! I'd forgotten I'd made this update. I have to set it to python3 in order for it to run on my machine.